data/scripts/sol-flow-node-type-gen.py.in

```diff
@@ -258,6 +258,7 @@ data_type_to_c_map = {
     "direction-vector": "struct sol_direction_vector",
     "string": "const char *",
     "location": "struct sol_location",
+    "timestamp": "time_t",
```

Comment:
From my pov we won't have problems related to timezone. It would receive it from other parts, for example, on JSON, and it would convert this time to time_t considering the timezone. When we need to convert it to other format that could be exposed (let's say, do something and convert it back to a string), it can be converted to whatever timezone we want, right? Internally we would always work with current timezone. As I see, we're good about timezone issues.

Regarding precision, indeed I only cared about seconds. None of these used functions / structs take less than seconds on consideration... So we would need to re-implement everything or look for a lib that handle it. I would let it go for now.